docs/changelog.md

```
@@ -1,3 +1 @@
-{%
-include-markdown "../CHANGELOG.md"
-%}
+--8<-- "CHANGELOG.md"
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)